web/components/TeamList.js

```diff
@@ -58,9 +45,9 @@ const mapStateToProps = ({ environment }) => {
     };
 };
 
-const mapDispatchToProps = () => {
+const mapDispatchToProps = (dispatch) => {
     return {
-        fetchAllTeamMembers
+        fetchAllTeamMembers: () => dispatch(fetchAllTeamMembers())
```

Comment:
Did you run into problems with the previous notation? The dispatch object shouldn't be necessary anymore in the latest Redux versions (again, the setlife templates are outdated so avoid using those as a standard)

Reply:
Yes, I had problems with the previous notation, only works for me with this. I know that in the latest redux version should not be necessary but didn't work, I dont know why.

Reply:
Ok I'll look into it